.github/workflows/repatriate.yml

```diff
@@ -0,0 +1,58 @@
+name: repatriate-to-lts
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  repatriate-to-lts:
+    runs-on: ubuntu-latest
+    name: repatriate-to-lts
+    env:
+      REF: ${{ github.ref }}
+      RUN: ${{ github.run_id }}
+      PUSH_ID: ${{ github.event.push_id }}
+      GH_TAIL_REF: ${{ github.event.before }}
+      GIT_API_KEY: ${{ secrets.GH_API_KEY }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GIT_USER_NAME: ${{ secrets.GH_AUTO_USER }}
+      GIT_USER_EMAIL: ${{ secrets.GH_AUTO_EMAIL }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Make repatriate branch
+        shell: bash
+        run: |
+          set -x
+          git config --global user.email "$GIT_USER_EMAIL"
+          git config --global user.name "$GIT_USER_NAME"
+          git config --global "url.https://$GIT_API_KEY@github.com.insteadof" https://github.com
+          git checkout -b ci-repatriate-changes-${RUN} origin/release/v1.14
+          git cherry-pick ${GH_TAIL_REF}..${GITHUB_REF} --allow-empty
+          git push --set-upstream origin ci-repatriate-changes-${RUN}
+      - name: Repatriate branch PR
+        shell: bash
+        run: |
+          set -x
+          echo "${GITHUB_TOKEN}" > temp_credential
+          # GH does NOT want this in the env
+          export GITHUB_TOKEN=
+          cat temp_credential | gh auth login --with-token
+          rm temp_credential
+          gh pr create --title "[CI] repatriate changes job ${RUN}" \
+                       --body "This PR contains changes from push to master ${REF}" \
+                       --base release/v1.14 \
+                       --reviewer "kflynn,aidanhahn,lukeshu,anodelman"
```

Comment:
This should just be `--reviewer emissary-ingress/emissary-maintainers` I think...

Reply:
Per @aidanhahn, using the list blows up. WTFO?